src/server/modes/charts/plugins/ql/yagr.ts

```diff
@@ -68,7 +68,8 @@ export default ({shared}: {shared: QlConfig}) => {
     });
 
     const isLegendEnabled = Boolean(
-        config.colors?.length && config.extraSettings?.legendMode !== 'hide',
+        (config.colors?.length && config.extraSettings?.legendMode !== 'hide') ||
+            config.extraSettings?.legendMode !== 'show',
```

Comment:
It looks like the mistake in the condition. isLegendEnabled and `legendMode !== 'show'` seems controversial